### DISCUSSION TITLE
bug  in some templates ?

### DISCUSSION THREAD

**Discussion by savushkin-yauheni** [projectdiscovery/nuclei-templates#5630] [General]
Hi team

After last nuclei update ( 2.7.8 ) some templates were broken. and `validate` option doesn't work on them.

They are:

```
vulnerabilities/other/nginx-module-vts-xss.yaml
vulnerabilities/oracle/oracle-ebs-xss.yaml
vulnerabilities/generic/basic-xss-prober.yaml
```

You can check it:

```
 nuclei -t vulnerabilities/generic/basic-xss-prober.yaml -target https://google.com --debug

Could not run nuclei: no valid templates were found

 nuclei -t vulnerabilities/generic/basic-xss-prober.yaml -target https://google.com --validate

All templates validated successfully
```

Thanks

**Comment by ehsandeep** [comment]
@savushkin-yauheni they have been excluded temporarily due to a known false positive issue https://github.com/projectdiscovery/nuclei-templates/issues/5405 in [.nuclei-ignore](https://github.com/projectdiscovery/nuclei-templates/blob/master/.nuclei-ignore)

**Comment by olearycrew** [comment]
Being tracked in #5405
